CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `time_convert` function
 ### Changed
 - Menu optimized
+- Test system modified
```

Comment:
Is this change important for user?

Reply:
I think it would be good if we mention it like this.